checklog-odoo.cfg

```diff
@@ -0,0 +1,3 @@
+[checklog-odoo]
+ignore=
+    WARNING.*produced a "xlsx" chunk, cannot examine it*
```

Comment:
Message comes from [this code](https://github.com/chaule97/mis-builder/blob/18.0-mig-mis_builder/mis_builder/tests/test_mis_report_instance.py#L505). And the reason is [try_report](https://github.com/odoo/odoo/blob/18.0/odoo/tools/test_reports.py#L63) returned warning because of xlsx format of report. You can use **`assertLogs`** to catch log and do log comparison here. We should not use odoo-checklog in this case.